pandas/io/excel.py

```diff
@@ -21,7 +21,7 @@
 from pandas.io.common import (_is_url, _urlopen, _validate_header_arg,
                               get_filepath_or_buffer, _NA_VALUES,
                               _stringify_path)
-from pandas.core.indexes.period import Period
+# from pandas.core.indexes.period import Period
```

Comment:
huh?

Reply:
Import no longer necessary because the elif on line 787 was folded into the else.  I'll remove it instead of commenting.